src/db_impl_gc.cc

```diff
@@ -44,6 +43,7 @@ void TitanDBImpl::BackgroundCallGC() {
     }
 
     bg_gc_scheduled_--;
+    MaybeScheduleGC();
```

Comment:
why do schedule here?

Reply:
this point is at the end of a GC, if there are some tasks in the queue, they will be scheduled, or if there is no task or the bg_thread run out nothing will happen.